docs/data/toolpad/components/button.md

```diff
@@ -0,0 +1,55 @@
+# Button
+
+<p class="description">Learn about a button component and it's usage in Toolpad.</p>
+
+Buttons allow users to take actions, make choices with a single tap. Detailed documentation of the component properties is available in the reference section for [button](/toolpad/reference/components/button/#properties).
+
+{{"demo": "ButtonBasic.js", "hideToolbar": true}}
+
+## Variations
```

Comment:
I think that in this screenshot most people won't understand it right away, because the descriptions are to the right of each variation.
Maybe we can use subheadings instead as the title for each variation, on top of it instead of to the right?

Reply:
I have removed the image, but the preview hasn't been updated as there is a merge conflict in the PR. I'll fix it today.